CONTRIBUTING.md

```diff
@@ -646,192 +642,200 @@ If you have any problems with formatting, please ping the [formatting team](http
 
-When in any doubt, please ask via a comment in your PR or through one of the help channels.
+When in any doubt, please ask via comment or through one of the help channels.
 
-## I received a review on my PR, how do I get it over the finish line?
+## I received a review, how do I get it over the finish line?
 
-In the review process, the committer will have left some sort of feedback on your PR.
-They may have immediately approved of your PR or even merged it but the more likely case is that they want you to change a few things or that they require further input.
+Most likely, a reviewer wants you to change a few things or requires further input.
 
-A reviewer may have taken a look at the code and it looked good to them ("Diff LGTM") but they still need to be convinced of the artefact's quality.
-They might also be waiting on input from other users of the component or its listed maintainer on whether the intention of your PR makes sense for the component.
+A reviewer may have taken a look at the code and it looked good to them ("Diff LGTM"), but they still need to be convinced of the artifact's quality.
+They might also be waiting on input from other users or its maintainer on whether the intention of your PR makes sense.
 If you know of people who could help clarify any of this, please bring the PR to their attention.
 The current state of the PR is frequently not clearly communicated, so please don't hesitate to ask about it if it's unclear to you.
 
-It's also possible for the reviewer to not be convinced that your PR is necessary or that the method you've chose to achieve your intention is the right one.
+It's also possible for the reviewer to not be convinced that your PR is necessary or that the method you've chosen is the right one.
 
 Please explain your intentions and reasoning to the committer in such a case.
 There may be constraints you had to work with which they're not aware of or qualities of your approach that they didn't immediately notice.
-(If these weren't clear to the reviewer, that's a good sign you should explain them in your commit message or code comments!)
+If these weren't clear to the reviewer, that's a good sign you should explain them in your commit message or code comments!
 
-There are some further pitfalls and realities which this section intends to make you aware of.
+There are some further pitfalls and realities to be aware of:
 
 ### Aim to reduce cycles
 
-Please be prepared for it to take a while before the reviewer gets back to you after you respond.
-This is simply the reality of community projects at the scale of Nixpkgs.
-As such, make sure to respond to _all_ feedback, either by applying suggested changes or argue in favor of something else or no change.
-It wastes everyone time waiting for a couple of days just for the reviewer to remind you to address something they asked for.
+It will take a while for the reviewer to get back to you after you respond.
```

Comment:
```suggestion
It will usually take a while for the reviewer to get back to you after you respond.
```

or
```suggestion
You should expect it to take a while for the reviewer to get back to you.
```

or

```suggestion
Please be prepared for it to take a while for the reviewer to get back to you, after you respond.
```
or

```suggestion
Be prepared for it to take a while for the reviewer to get back to you after you respond.
```

?